src/py42/modules/alertrules.py

```diff
@@ -84,14 +84,27 @@ def get_all(self, sort_key="CreatedAt", sort_direction="DESC"):
         alerts_client = self.microservice_client_factory.get_alerts_client()
         return alerts_client.get_all_rules(sort_key=sort_key, sort_direction=sort_direction)
 
-    def get_by_name(self, rule_name):
+    def get_all_by_name(self, rule_name):
```

Comment:
This is a breaking change for a couple reasons:
- method name change
- returns generator instead of list

However, I'm okay with it because it it was very recently introduced, nothing we know of is using it, and and it makes the API more concistent.